code-challenges.test.js

```diff
@@ -14,6 +14,24 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Psuedocode
+// Write out a describe statemtn that contains the function fibanacci
+// create an it statement describing that itll take in the variable and put out a sequence.
+// create two expect statements using .toEqual to tell the test what the output should equal 
+
+describe ("fibanacci", () => {
+  it ("should return the expected fibanacci seuquence", () => {
+    const fibLength1 = 6
+    const fibLength2 = 10
+      expect (fib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
```

Comment:
You have a couple of different function names in your test. We want them to be the same so we don't run into incorrect failures